### PR TITLE
5.4.0 changelog

### PR DESCRIPTION
**Pull request by ashnwade**
This PR addresses no issue. This is the changelog for the v5.4.0 release. 
I did not include bugs that were introduced and fixed within this development cycle.

### CODE REVIEW

File: changelog/5.4.0.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+* Improved behavior when dealing with large attachments in Email flow node.
+* Improved handling of webserver startup when datastore is unreachable.
+* Improved handling of loadbalancer startup when datastore is unreachable.
+* Improved performance for Ingest flow node.
+
+## Ingester Changes
+
+### Bug Fixes
+
+* Fixed an issue that prevented an ingester from exiting if it got into a bad state with both disk and cache being full.
```
Comment (kris-watts-gravwell) [inline changelog/5.4.0.md:85]: Add `All production ingesters now support a global Attach directive to attach arbitrary intrinsic enumerated values.`

**Review by kris-watts-gravwell — CHANGES_REQUESTED** [review]
one more item in ingester features

**Comment by kris-watts-gravwell** [comment]
we should probably call out the runtime upgrade to go 1.21 too

**Review by michael-wisely-gravwell — COMMENTED** [review]
UI stuff LGTM :+1:

**Review by kris-watts-gravwell — APPROVED** [review]
LGTM

**Pull request merged**